deployment/docker-compose/template.core_backend.env

```diff
@@ -75,4 +75,5 @@ LANGFUSE=False
 # LANGFUSE_HOST="https://cloud.langfuse.com"
 
 ##### Google Cloud Storage Variables#############################################
-# BUCKET_NAME="aaq-speech-test"
+# GCP_SPEECH_BUCKET="aaq-speech-test"
```

Comment:
@MustafaAkolawala I think this should be `GCS` not `GCP`

Reply:
AAH a silly mistake my bad